draft-irtf-cfrg-frost.md

```diff
@@ -484,28 +496,32 @@ This section describes the subroutine for computing binding factors based
 on the participant commitment list and message to be signed.
 
 ~~~
-  Inputs:
-  - commitment_list = [(i, hiding_nonce_commitment_i, binding_nonce_commitment_i), ...],
-    a list of commitments issued by each participant, where each element in the list
-    indicates a NonZeroScalar identifier i and two commitment Element values
-    (hiding_nonce_commitment_i, binding_nonce_commitment_i). This list MUST be sorted
-    in ascending order by identifier.
-  - msg, the message to be signed.
-
-  Outputs:
-  - binding_factor_list, a list of (NonZeroScalar, Scalar) tuples representing the binding factors.
-
-  def compute_binding_factors(commitment_list, msg):
-    msg_hash = H4(msg)
-    encoded_commitment_hash = H5(encode_group_commitment_list(commitment_list))
-    rho_input_prefix = msg_hash || encoded_commitment_hash
-
-    binding_factor_list = []
-    for (identifier, hiding_nonce_commitment, binding_nonce_commitment) in commitment_list:
-      rho_input = rho_input_prefix || G.SerializeScalar(identifier)
-      binding_factor = H1(rho_input)
-      binding_factor_list.append((identifier, binding_factor))
-    return binding_factor_list
+Inputs:
+- commitment_list = [(i, hiding_nonce_commitment_i,
+  binding_nonce_commitment_i), ...], a list of commitments issued by
+  each participant, where each element in the list indicates a
+  NonZeroScalar identifier i and two commitment Element values
+  (hiding_nonce_commitment_i, binding_nonce_commitment_i). This list
+  MUST be sorted in ascending order by identifier.
+- msg, the message to be signed.
+
+Outputs:
+- binding_factor_list, a list of (NonZeroScalar, Scalar) tuples
+  representing the binding factors.
+
+def compute_binding_factors(commitment_list, msg):
+  msg_hash = H4(msg)
+  encoded_commitment_hash = H5(
+      encode_group_commitment_list(commitment_list))
```

Comment:
```suggestion
  encoded_commitment_hash =
      H5(encode_group_commitment_list(commitment_list))
```

Reply:
In Python this does not work because the syntax is ambiguous; it requires escaping the newline with `\`.

It works in the committed version because the single `(` makes the parser understand that the statement continues in another line.

Of course we don't need to be fully compliant with Python, so I leave the decision to you.

Reply:
It _feels_ better to not have orphan function calls like this, so I'd prefer we go with the suggestion, despite it not being valid Python.